apps/balancer-tools/src/app/apr/(components)/MultiSelectDropdown.tsx

```diff
@@ -111,7 +113,7 @@ export const MultiSelectDropdown = ({
                         toggleMenu();
                       }}
                       onKeyDown={handleOnKeyDown}
-                      className="hover:shadow-[0] focus:shadow-[0] shadow-none flex-1"
+                      className="hover:shadow-[0] focus:shadow-[0] shadow-none flex-1 !h-6"
```

Comment:
🤔 why did we remove the shadows here? Also curious about why the height wasn't applying and needed a `!` here

Reply:
This component has the class `h-[35px]` was being applied, I added important to overwrite it